lib/alice.ex

```diff
@@ -30,23 +35,28 @@ defmodule Alice do
   defp children(:test, _), do: []
 
   defp children(_env, extras) do
-    import Supervisor.Spec, warn: false
-
-    state_backend_children() ++
-      [
-        worker(Alice.Router, [handlers(extras)]),
-        worker(Alice.ChatBackends.Slack, [])
-      ]
+    state_backend_children() ++ router(extras) ++ chat_backend()
```

Comment:
maybe call these `router_children` & `chat_backend_children` or rename `state_backend_children` to just `state_backend`, though "children" is a common term for this kind of thing in supervisors

Reply:
Done!